Bottom: 41d8934c86c931a8568293264d1b3c6b80101666
Top:    bfc5f2ae62edbdd95f1c42fdcc53bf54fb120fd0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-18 02:15:38 +0000

Refresh of find-h-annotate-which-exported

---

diff --git a/src/find.h b/src/find.h
index de0a8f8..b04927f 100644
--- a/src/find.h
+++ b/src/find.h
@@ -47,7 +47,7 @@ bool LineLineIntersect (LineType *, LineType *);
 bool LineArcIntersect (LineType *, ArcType *);
 bool PinLineIntersect (PinType *, LineType *);
 bool LinePadIntersect (LineType *, PadType *);
-bool ArcPadIntersect (ArcType *, PadType *);
+/* NOT USED EXTERNALLY */// bool ArcPadIntersect (ArcType *, PadType *);
 void LookupElementConnections (ElementType *, FILE *);
 void LookupConnectionsToAllElements (FILE *);
 void LookupConnection (Coord, Coord, bool, Coord, int, bool AndRats);
@@ -56,11 +56,11 @@ bool ResetFoundLinesAndPolygons (bool, int flag);
 bool ResetFoundPinsViasAndPads (bool, int flag);
 bool ResetConnections (bool, int flag);
 void InitConnectionLookup (void);
-void InitComponentLookup (void);
-void InitLayoutLookup (void);
+/* NOT USED EXTERNALLY */// void InitComponentLookup (void);
+/* NOT USED EXTERNALLY */// void InitLayoutLookup (void);
 void FreeConnectionLookupMemory (void);
-void FreeComponentLookupMemory (void);
-void FreeLayoutLookupMemory (void);
+/* NOT USED EXTERNALLY */// void FreeComponentLookupMemory (void);
+/* NOT USED EXTERNALLY */// void FreeLayoutLookupMemory (void);
 void RatFindHook (int, void *, void *, void *, bool, int flag, bool);
 int DRCAll (void);
